src/main/java/jp/skypencil/errorprone/slf4j/VariableGenerator.java

```diff
@@ -0,0 +1,87 @@
+package jp.skypencil.errorprone.slf4j;
+
+import com.google.common.base.CaseFormat;
+import com.google.common.collect.Ordering;
+import com.google.errorprone.VisitorState;
+import com.google.errorprone.fixes.Fix;
+import com.google.errorprone.fixes.SuggestedFix;
+import com.sun.source.tree.AnnotationTree;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Use classes from the Java API instead of Sun classes. ([squid:S1191](https://sonarcloud.io/organizations/kengotoda-github/rules#rule_key=squid%3AS1191))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=jp.skypencil.errorprone.slf4j%3Aerrorprone-slf4j&pullRequest=4&issues=AWRd1zxl1IJZDfsS72ji&open=AWRd1zxl1IJZDfsS72ji)

[ISSUE_KEY]: AWRd1zxl1IJZDfsS72ji